release/kata-deploy-binaries.sh

```diff
@@ -167,6 +167,18 @@ install_firecracker() {
 	popd
 }
 
+# Install static cloud-hypervisor asset
+install_cloud_hypervisor() {
+	info "build static cloud-hypervisor"
+	"${script_dir}/../static-build/cloud-hypervisor/build-static-clh.sh"
+	info "Install static cloud-hypervisor"
+	mkdir -p "${destdir}/opt/kata/bin/"
+	sudo install -D --owner root --group root --mode 0744 cloud-hypervisor/cloud-hypervisor "${destdir}/opt/kata/bin/cloud-hypervisor"
+	pushd "${destdir}"
+	tar -czvf ../kata-cloud-hypervisor-static.tar.gz *
```

Comment:
I'm presuming this is the common idiom for the release method - the tarfile - but, maybe add a comment, as it looks slightly odd building everything into a nice binary dir, and then tarring it up. Just a comment like `# tar up with the final destination path in place so the install can just untar in situ` or something?

Reply:
Added a comment, let me  know if is clear enough